src/CanvasDownloadLinks.js

```diff
@@ -8,26 +8,24 @@ import uniqBy from 'lodash/uniqBy';
 import { OSDReferences } from 'mirador/dist/es/src/plugins/OSDReferences';
 import RenderingDownloadLink from './RenderingDownloadLink';
 
-
 /**
  * CanvasDownloadLinks ~
 */
 export default class CanvasDownloadLinks extends Component {
   zoomedImageLabel() {
+    const { t } = this.props;
     const bounds = this.currentBounds();
-    return `Zoomed region (${Math.floor(bounds.width)} x ${Math.floor(bounds.height)}px)`;
+    return `${t('zoomed_region')} (${Math.floor(bounds.width)} x ${Math.floor(bounds.height)}px)`;
```

Comment:
We'll probably want to i18n the whole pattern here and below (especially for RTL languages, where the order might be different). I think the javascript i18next library does some fancy things with interpolation, maybe something like:

```
t('download_link', { name: '{{zoomed_region}}', width: Math.floor(bounds.width), height: Math.floor(bounds.height) })
```

https://www.i18next.com/translation-function/interpolation